src/app/features/balances-list/components/stacks-assets-fungible.tsx

```diff
@@ -0,0 +1,30 @@
+import { Suspense } from 'react';
+import { Stack, StackProps } from '@stacks/ui';
+import { StacksAssetFtRow } from '@app/components/assets/stacks/stacks-asset-ft-row';
+import { useAssetWithMetadata, useStacksFungibleTokensState } from '@app/store/assets/assets.hooks';
+
+import { StacksAssetFt } from '@shared/models/asset.model';
+
+interface FungibleAssetRowProps {
+  asset: StacksAssetFt;
+}
+function FungibleAssetRow({ asset }: FungibleAssetRowProps) {
+  const assetWithMeta = useAssetWithMetadata(asset);
+  return <StacksAssetFtRow asset={assetWithMeta} />;
+}
+
+export function StacksAssetsFungible(props: StackProps) {
+  const fungibleTokens = useStacksFungibleTokensState();
+
+  if (!fungibleTokens) return null;
+
+  return (
+    <Stack spacing="loose" {...props}>
+      {fungibleTokens?.map(asset => (
```

Comment:
Already checked for null

```suggestion
      {fungibleTokens.map(asset => (
```